include/tlog/errs.h

```diff
@@ -141,4 +141,68 @@ extern void tlog_errs_pushf(struct tlog_errs **perrs, const char *fmt, ...)
  */
 extern int tlog_errs_print(FILE *stream, const struct tlog_errs *errs);
 
+/**
+ * Push an error to the perrs stack, with message as a constant string
+ * and then go to cleanup.
+ *
+ * @param fmt   The message string to copy and put into the pushed error.
```

Comment:
The "fmt" name here is wrong, we're not passing a format string here, but just a message, so it should be "msg" or such instead. Also, please make sure that the leading underscore is present and argument names in the description and in the macro definition match.